Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -18,16 +18,24 @@ internal static class ProfileManager
     {
         private const int ProcessExitTimeoutInSeconds = 180;
 
+        private const string UserModeCustomProviderAgentGuid = "F5091AA9-80DC-49FF-A7CF-BD1103FE149D";
+        private const string DetailedTracingAgentGuid = "1C92BA2A-A990-480F-A02F-40068871CAAC";
+        private const string IISWWWServerProviderGuid = "3A2A4E84-4C21-4981-AE10-3FDA0D9B0F83";
```

Comment:
nit: use standard pascal casing for acronyms longer than 2 characters. this better be called IisWebServerProviderGuid